app/resources/amplanguage-menu.html.template

```diff
@@ -0,0 +1,44 @@
+{# Copyright 2010 Google Inc.  Licensed under the Apache License, Version   #}
+{# 2.0 (the "License"); you may not use this file except in compliance with #}
+{# the License.  You may obtain a copy of the License at:                   #}
+{#     http://www.apache.org/licenses/LICENSE-2.0                           #}
+{# Unless required by applicable law or agreed to in writing, software      #}
+{# distributed under the License is distributed on an "AS IS" BASIS,        #}
+{# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. #}
+{# See the License for the specific language governing permissions and      #}
+{# limitations under the License.                                           #}
+
+{# Template fragment for the menu of available languages.                   #}
+
+{% load i18n %}
+
+<div class="languages" title="{% trans "Language selection" %}">
```

Comment:
This file looks very similar to language-menu.html.template, so ideally we don't want to duplicate the whole file. But I do see some difference. Some ideas:

* Use a single template file, but use {% if env.amp %} for parts which differ.
* Extract the content of the SELECT tag as a sub template file because that part looks exactly the same.